Apps/Medication/src/appsettings.Development.json

```diff
@@ -13,13 +13,38 @@
         "ClientId": "healthgateway-local"
     },
     "HNClient": {
-        "Url": "https://hnc-dev.healthgateway.gov.bc.ca/"
+        "Url": "https://hnc-dev.healthgateway.gov.bc.ca/",
+        "SendingApplication": "GATEWAY",
+        "SendingFacility": "BC01001242",
+        "ReceivingApplication": "PNP",
+        "ReceivingFacility": "CPA",
+        "MessageVersion": "2.1",
+        "ProcessingID": "D",
+        "ZCA": {
+            "BIN": "1",
+            "CPHAVersionNumber": "70",
+            "TransactionCode": "00",
+            "SoftwareId": "HG",
+            "SoftwareVersion": "01"
+        },
+        "ZCB": {
+            "PharmacyId": "BCXX000034"
+        },
+        "ZZZ": {
+            "PractitionerId": "XXAPZ",
+            "PractitionerIdRef": "91"
+        },
+        "ZPL": {
+            "TransactionReasonCode": "MM"
+        },
+        "UnixTimeZoneId": "America/Vancouver",
+        "WindowsTimeZoneId": "Pacific Standard Time"
     },
     "AuthService": {
         "TokenUri": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/ff09qn3f/protocol/openid-connect/token",
-        "Audience": "healthgateway-local",
-        "ClientId": "medication-service",
-        "ClientSecret": "f8abee65-8711-4092-8260-a0a934de66b9"
+        "Audience": "medication-service",
+        "ClientId": "*",
```

Comment:
Should this be removed?

Reply:
+1. I believe it should. The CORS section is handled on the common code

Reply:
the client secret needs to be a configuration setting for the environments. Each KeyCloak env will have its own secret for the Medication Service to use to authenticate as a client-credentials-grant 'system account' in order to obtain the right token for accessing HNClient, I have removed the secret so it's published.